packages/replayio/src/utils/spawnProcess.ts

```diff
@@ -29,30 +34,28 @@ export function spawnProcess(
     // github.com/replayio/replay-cli/pull/344#discussion_r1553258356
     spawned.unref();
   } else {
+    const stderr = collectData(spawned.stderr);
```

Comment:
this was lost as we were never listening to `stderr` so we had no chance to surface what was put there by the launched browser

Reply:
Kind of a nit but why a separate method for this? It's only ever referenced once, and the other process event handlers are inline methods.

Reply:
no particular reason, it could be reused to collect `stdout` at some point but we don't need it rn